From: Yun-Ze Li <p76091292@gs.ncku.edu.tw>
Subject: mm, docs: fix comments that mention mem_hotplug_end()
Date: Mon, 20 Jun 2022 07:15:16 +0000

Comments that mention mem_hotplug_end() are confusing as there is no
function called mem_hotplug_end().  Fix them by replacing all the
occurences of mem_hotplug_end() in the comments with mem_hotplug_done().

Link: https://lkml.kernel.org/r/20220620071516.1286101-1-p76091292@gs.ncku.edu.tw
Signed-off-by: Yun-Ze Li <p76091292@gs.ncku.edu.tw>
Cc: Souptick Joarder <jrdr.linux@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/mmzone.h |    4 ++--
 mm/compaction.c        |    2 +-
 mm/vmscan.c            |    2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

--- a/include/linux/mmzone.h~mm-docs-fix-comments-that-mention-mem_hotplug_end
+++ a/include/linux/mmzone.h
@@ -595,7 +595,7 @@ struct zone {
 	 * give them a chance of being in the same cacheline.
 	 *
 	 * Write access to present_pages at runtime should be protected by
-	 * mem_hotplug_begin/end(). Any reader who can't tolerant drift of
+	 * mem_hotplug_begin/done(). Any reader who can't tolerant drift of
 	 * present_pages should get_online_mems() to get a stable value.
 	 */
 	atomic_long_t		managed_pages;
@@ -874,7 +874,7 @@ typedef struct pglist_data {
 	unsigned long nr_reclaim_start;	/* nr pages written while throttled
 					 * when throttling started. */
 	struct task_struct *kswapd;	/* Protected by
-					   mem_hotplug_begin/end() */
+					   mem_hotplug_begin/done() */
 	int kswapd_order;
 	enum zone_type kswapd_highest_zoneidx;
 
--- a/mm/compaction.c~mm-docs-fix-comments-that-mention-mem_hotplug_end
+++ a/mm/compaction.c
@@ -3030,7 +3030,7 @@ void kcompactd_run(int nid)
 
 /*
  * Called by memory hotplug when all memory in a node is offlined. Caller must
- * hold mem_hotplug_begin/end().
+ * be hold mem_hotplug_begin/done().
  */
 void kcompactd_stop(int nid)
 {
--- a/mm/vmscan.c~mm-docs-fix-comments-that-mention-mem_hotplug_end
+++ a/mm/vmscan.c
@@ -4645,7 +4645,7 @@ void kswapd_run(int nid)
 
 /*
  * Called by memory hotplug when all memory in a node is offlined.  Caller must
- * hold mem_hotplug_begin/end().
+ * be hold mem_hotplug_begin/done().
  */
 void kswapd_stop(int nid)
 {
_
